pkg/consensus/crypto.go

```diff
@@ -15,12 +15,11 @@ type privateKey struct {
 
 // MarshalBinary implements the encoding.BinaryMarshaler interface.
 func (p privateKey) MarshalBinary() (data []byte, err error) {
-	return p.PrivateKey.Bytes(), nil
+	return nil, nil
```

Comment:
Then maybe it's safer to use `panic("Not implemented")` or something like this?

Reply:
Doesn't matter, this will be removed after dbft upgrade.

Reply:
And it is now.